test/test_retriever.py

```diff
@@ -681,13 +688,22 @@ def test_reset_retriever(tmpdir):
     pwd_name = os.getcwd()
     workdir = tmpdir.mkdtemp()
     workdir.chdir()
-    dataset = random.choice(rt.dataset_names())
+    offline_datasets = rt.dataset_names()['offline']
+    test_datasets = ['test-eco-level-four', 'test-raster-bio1', 'test-raster-bio2',
+     'test-raster-multi-bands', 'test-raster-single-band', 'test-us-eco', 'test-vector']
+    offline_datasets = [dataset for dataset in offline_datasets if dataset not in test_datasets]
```

Comment:
> I think this happened, becausetest_reset_retriever picks a random dataset from the offline scripts and deletes it. In the above case, that random dataset happened to be test-raster-bio2, which is not available upstream. Thus, after deletion it could not be downloaded again, causing the tests to fail.

@harshitbansal05, how about 
```python
offline_datasets = [dataset for dataset in offline_datasets if not dataset.startswith("test-")]
```

Reply:
yes, this would be good. Thanks @henrykironde.